python/cudf_polars/tests/testing/test_asserts.py

```diff
@@ -26,7 +26,7 @@ def test_translation_assert_raises():
     class E(Exception):
         pass
 
-    unsupported = df.group_by("a").agg(pl.col("a").cum_max().alias("b"))
+    unsupported = df.group_by("a").agg(pl.col("a").upper_bound().alias("b"))
```

Comment:
This is fine for now since it's already doing this just with another function, but I'll do some thinking about how we can test the IR translation utility without needing to chase an actually unsupported feature